src/main/java/reposense/model/Author.java

```diff
@@ -144,6 +144,20 @@ public void appendIgnoreGlobList(List<String> ignoreGlobList) {
         updateIgnoreGlobMatcher();
     }
 
+    /**
+     * Validates and adds a list of ignore glob into the {@code Author} class instance variable without duplicates
+     * and updates the ignore glob matcher.
+     */
+    public void appendIgnoreGlobListWithoutDuplicates(List<String> ignoreGlobList) {
```

Comment:
Can this method replace `Author#appendIgnoreGlobList`?

Reply:
Can replace but I am not very sure if you want to keep the original one. 
 Compared to `appendWithoutDuplicates`, `Author#appendIgnoreGlobList` saves the time looping and searching for duplicates before adding, so it has better performance if we can make sure no duplicates exist, like the case in `AuthorConfiguration#update` which uses
`Author#appendIgnoreGlobList`.

Reply:
If there is no clear advantage in keeping the `appendIgnoreGlobList`, it will be better to replace it since the processing speed of this is marginal. 

The new method can be named `importIgnoreGlobList` instead